### PR TITLE
Update product-management-team-handbook.md

### PR DESCRIPTION
**Pull request by wiersgallak**

### CODE REVIEW

**Comment by wiersgallak** [comment]
The section we added it to was not immediately navigatable.  Adding to a few main pages.

File: operations/research-and-development/product/product-management-team-handbook.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ description: 1% DRAFT
 
 The purpose of this handbook is to provide more details on the Product Manager role, processes, and relations to other functions within the organization to foster success for employees in this role. This handbook may be shared with other functional areas so other employees may gain a better understanding of the Product Manager role, thereby having a greater understanding for responsibilities and interaction areas.
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-management-team-handbook.md]:
```suggestion
The purpose of this handbook is to provide more details on the Product Manager role, processes, and relations to other functions within the organization to foster success for employees in this role. This handbook may be shared with other functional areas so they may gain a better understanding of the Product Manager role, and develop a greater understanding for responsibilities and interaction areas.
```

**Review by justinegeffen — COMMENTED** [review]
Just one minor comment, otherwise this is great. :) I like the updated README!

**Review by cwarnermm — DISMISSED** [review]
Reads well and I agree with inline editorial feedback.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**